src/modules/facility-activation-transaction/facility-activation-transaction.controller.ts

```diff
@@ -0,0 +1,70 @@
+import { Body, Controller, Param, ParseArrayPipe, Post } from '@nestjs/common';
+import {
+  ApiBadRequestResponse,
+  ApiBody,
+  ApiCreatedResponse,
+  ApiInternalServerErrorResponse,
+  ApiNotFoundResponse,
+  ApiOperation,
+  ApiParam,
+} from '@nestjs/swagger';
+import { EXAMPLES } from '@ukef/constants';
+import { FacilityService } from '@ukef/modules/facility/facility.service';
+
+import {
+  CreateFacilityActivationTransactionRequest,
+  CreateFacilityActivationTransactionRequestItem,
+} from './dto/create-facility-activation-transaction-request.dto';
+import { CreateFacilityActivationTransactionResponse } from './dto/create-facility-activation-transaction-response.dto';
+import { FacilityActivationTransactionParamsDto } from './dto/facility-activation-transaction-params.dto';
+import { FacilityActivationTransactionService } from './facility-activation-transaction.service';
+
+@Controller()
+export class FacilityActivationTransactionController {
+  constructor(private readonly facilityActivationTransactionService: FacilityActivationTransactionService, private readonly facilityService: FacilityService) {}
+
+  @Post('facilities/:facilityIdentifier/activation-transactions')
```

Comment:
you probably have this covered already, but remember to update the Confluence page to reflect the changes

Reply:
Added it now.